cegs_portal/static/css/project.css.tw

```diff
@@ -55,6 +55,10 @@
 
 
 
+
```

Comment:
I think you can remove this extra white space.

Reply:
Completed